e2e/testdata/snapshot_restore/snapshot-template.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: snapshot.storage.k8s.io/v1
+kind: VolumeSnapshot
+metadata:
+  name: %s
+spec:
+  volumeSnapshotClassName: topolvm-provisioner-thin
```

Comment:
If this is hardcoded, why does the snapshotclass.yaml need the name to be specified?

Reply:
Nice catch! Snapshot template will also take snapclass as an input parameter